### DISCUSSION TITLE
Option to search duplicates between two folders, but not duplicates inside the same folder

### DISCUSSION THREAD

**Discussion by liam-maps** [qarmin/czkawka#497] [Ideas]
Hello.
Sometimes I'd like to compare two folders and keep only one file, because I'm sure that there's going to be at least one duplicate on each folder I select for comparison.
I'd like to avoid having matches inside the same folder, and that's something that the program warns about, but sometimes I'd like this behaviour:

> If there's one match in folder A and one match in folder B, keep one of them based on file path, and I'd like to avoid having multiple matches inside one folder and deleting all of them.

**Comment by qarmin** [comment]
It is implemented by https://github.com/qarmin/czkawka/pull/516
